src/pe/import.rs

```diff
@@ -293,6 +298,32 @@ impl<'a> SyntheticImportDirectoryEntry<'a> {
             import_address_table,
         })
     }
+
+    pub fn get_imports_rva<T: Bitfield<'a>>(
+        &self,
+    ) -> error::Result<Vec<SyntheticImportAddressTableEntry<'a>>> {
+        let mut out = Vec::<SyntheticImportAddressTableEntry>::new();
```

Comment:
I don’t see any fallibility here, so I’d drop the error return. Also I would attempt to get rid of allocation via returning a vec and try `impl Iterator<Item = SyntheticImportAddressTableEntry<‘a>> + ‘a`

(you may have to play with the lifetimes). Since it’s optional too you may have to do `self.import_table.unwrap_or(&[]).iter().enumerate().flatmap`

Reply:
Got it will implement it as an iterator